02 Requiements/UseCases/UseCase2/CasualUC2.md

```diff
@@ -0,0 +1,40 @@
+# Use case 2: Customize Order
+
+**Actors:**  
+Consultant
+
+**Main scenario:**  
+The application prompts the user to specify additional components.  
+A Consultant specifies if the New Yorker wall has a door.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies which door type the New Yorker wall has.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies if the New Yorker wall has a Lockbox.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies if the New Yorker wall has a handle.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies which handle type the New Yorker wall has.  
+A Consultant specifies if the New Yorker wall has special glass.  
+&nbsp;&nbsp;&nbsp; **-** A Consultant specifies which special glass type the New Yorker wall has.  
+A Consultant specifies if the New Yorker wall is for a Wetroom.  
+A Consultant specifies if the New Yorker wall is a Shower wall.  
+The application redirects the Consultant to *Preview Order*.
+
+
+**Alternate scenarios**  
+The Consultant chooses to not customize the order.  
```

Comment:
Is this line needed, if nothing is done is it a choice?

Reply:
For the sake of understanding then i would say yes.